llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -234,6 +234,12 @@ cl::opt<bool> EmitOnlyKernelsAsEntryPoints{
              "device code split"),
     cl::cat(PostLinkCat), cl::init(false)};
 
+cl::opt<bool> SplitForRunTimeLinking{
+    "split-for-run-time-linking",
```

Comment:
Do we also need to hook up the driver/clang-linker-wrapper passing this, or will that be a separate PR?

Reply:
Changes made to driver to pass -exclude-external-functions if -shared seen.